src/overlays/actors/ovl_En_Door/z_en_door.h

```diff
@@ -4,14 +4,57 @@
 #include <ultra64.h>
 #include <global.h>
 
+/**
+ * Actor Parameters
+ * 
+ * |                  |       |             |
+ * | Transition Index | Type  | Double Door | Switch Flag OR Text Id - 0x0200
+ * |------------------|-------|-------------|---------------------------------
+ * | 0 0 0 0 0 0      | 0 0 0 | 0           | 0 0 0 0 0 0 
+ * | 6                | 3     | 1           | 6
+ * |
+ * 
+ * Transition Index     1111110000000000
+ * Type                 0000001110000000
+ * Double Door          0000000001000000
+ * Switch Flag          0000000000111111
+ * Text id - 0x0200     0000000000111111
+ * 
+ */
+
+typedef struct {
+    /* 0x00 */ s16 sceneNum;
+    /* 0x02 */ u8 displaylistIdx;
+    /* 0x04 */ s16 objectId;
+} EnDoorInfo;
+
+typedef enum {
+    /* 0x00 */ DOOR_0,          // loads rooms
+    /* 0x01 */ DOOR_LOCKED,     // small key locked door
+    /* 0x02 */ DOOR_1,          // loads rooms
+    /* 0x03 */ DOOR_NOLOAD,     // doesn't load rooms, used for doors paired with scene transition polygons
```

Comment:
NOLOAD is a bit confusing imo, makes me think about "don't load door"
maybe `DOOR_EXIT` / `DOOR_SCENE_EXIT` / `DOOR_EXIT_SCENE`? exit as in "scene exit list" (cf `func_800990F0`)

but actually, can this type be used for a same-room "transition"?